src/main/resources/api/service_openapi.yaml

```diff
@@ -90,8 +88,8 @@ paths:
           description: Success
         '400':
           $ref: '#/components/responses/BadRequest'
-        '401':
-          $ref: '#/components/responses/NotFound'
+        '403':
```

Comment:
FWIW If not found is the more common case, I'd probably prefer 404 to 403 where there's ambiguity.